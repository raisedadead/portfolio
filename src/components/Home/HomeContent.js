import React from 'react';
import { ClassNames } from '@emotion/core';

import { StaticQuery, graphql } from 'gatsby';
import Box from 'mineral-ui/Box';
import Text from 'mineral-ui/Text';
import Flex from 'mineral-ui/Flex';
import Tooltip from 'mineral-ui/Tooltip';
import Img from 'gatsby-image';

import LinkItem from '../Link';

const Link = props => (
  <ClassNames>
    {({ css }) => (
      <LinkItem
        className={css`
          outline: none;
          text-decoration: none;
          padding: 2px 1px 0;
          border-bottom-style: dashed;
          border-bottom-color: currentcolor;
          border-bottom-width: 1px;
          color: currentcolor;
        `}
        {...props}
      />
    )}
  </ClassNames>
);

export const PureHomeContent = ({ data }) => (
  <Box
    as="section"
    marginHorizontal="auto"
    breakpoints={[`narrow`, `medium`, `wide`]}
    width={[`80vw`, `80vw`, `95vw`, `50vw`]}
  >
    <Flex
      alignItems="center"
      justifyContent="center"
      direction="column"
      height="75vh"
    >
      <Box
        breakpoints={[`narrow`, `medium`, `wide`]}
        width={[`25vw`, `25vw`, `15vw`, `15vw`]}
        margin="1em"
      >
        <Img sizes={data.profileImage.childImageSharp.fluid} />
      </Box>
      <Text as="h2" align="center">
        <Tooltip cursor="help" content="/mru:dʒi:eʃ/" placement="top" usePortal>
          mrugesh
        </Tooltip>
        {` `}
        mohapatra
      </Text>

      <Text as="h3" align="center">
        developer. music addict. open source enthusiast. noob photographer.
      </Text>
      <br />
      <Text as="p" align="center">
        Lead Developer and Community Advocate at{`  `}
        <Link to="https://www.freecodecamp.org" aria-label="freecodecamp.org">
          freeCodeCamp.org
        </Link>
      </Text>
    </Flex>
  </Box>
);

export const HomeContent = () => (
  <StaticQuery
    query={graphql`
      query {
        profileImage: file(relativePath: { eq: "profile.png" }) {
          childImageSharp {
            fluid(maxWidth: 1000) {
              ...GatsbyImageSharpFluid_withWebp_tracedSVG
            }
          }
        }
      }
    `}
    render={data => <PureHomeContent data={data} />}
  />
);

export default HomeContent;
