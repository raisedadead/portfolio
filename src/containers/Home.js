import React, { Component } from 'react'
import {
  // MainNav,
  Footer,
  Background,
  StylesHelper,
  ImageLoader
} from '../components'

import HomeContent from './HomeContent'

const mainContainer = StylesHelper(
  'uk-container',
  'uk-cover-container',
  'uk-width-viewport',
  'uk-height-viewport',
  'uk-padding-remove'
)
const content = StylesHelper('uk-position-center', 'uk-text-center')

const contentSyles = {
  width: '60vw',
  display: 'flex',
  flexDirection: 'column',
  justifyContent: 'center',
  alignItems: 'center'
}
export default class Home extends Component {
  render() {
    return (
      <div className={mainContainer}>
        <Background data-uk-cover />
        <div
          className="uk-position-cover"
          style={{
            background: 'linear-gradient(to bottom, #36d1dc88, #5b86e5)'
          }}
        />
        <div style={contentSyles} className={content}>
          <div
            style={{ width: '15vw', height: '15vw', borderRadius: '50%' }}
            className="uk-box-shadow-large"
          >
            <ImageLoader src="assets/images/profile.png" alt="profile image" />
          </div>
          <HomeContent />
          {/*
          <MainNav />
          */}
        </div>
        <Footer />
      </div>
    )
  }
}
