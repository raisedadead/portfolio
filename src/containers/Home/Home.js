import React, { Component } from 'react'
import { TopNav, SocialNav, Footer, Background } from '../../components'
import HomeContent from './HomeContent'
import { StylesHelper } from '../../components/'

const mainContainer = StylesHelper(
  'uk-container',
  'uk-cover-container',
  'uk-width-viewport',
  'uk-height-viewport',
  'uk-padding-remove'
)
const content = StylesHelper('uk-position-center', 'uk-text-center')

export default class Home extends Component {
  render() {
    return (
      <div className={mainContainer}>
        <Background data-uk-cover />
        <TopNav />
        <div className={content}>
          <HomeContent />
          <SocialNav />
        </div>
        <Footer />
      </div>
    )
  }
}
