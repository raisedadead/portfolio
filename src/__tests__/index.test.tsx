import { render, screen } from '@testing-library/react';
import Home from '../pages/index';

describe('Home', () => {
  it('renders the the name on the homepage', () => {
    render(<Home />);
    expect(screen.getByText('mrugesh mohapatra')).toBeInTheDocument();
  });

  it('renders the the description on the homepage', () => {
    render(<Home />);
    expect(
      screen.getByText(
        'nocturnal developer 🦉 • open-source enthusiast 🌏 • photography noob 📷'
      )
    ).toBeInTheDocument();
  });

  it('renders the the social links on the homepage', () => {
    render(<Home />);
    expect(screen.getByText('GitHub')).toBeInTheDocument();
    expect(screen.getByText('Twitter')).toBeInTheDocument();
    expect(screen.getByText('LinkedIn')).toBeInTheDocument();
    expect(screen.getByText('Instagram')).toBeInTheDocument();
  });

  it('renders a snapshot of the homepage', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
