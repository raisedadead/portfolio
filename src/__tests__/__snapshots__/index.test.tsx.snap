// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a snapshot of the homepage 1`] = `
<DocumentFragment>
  <div
    class="fixed inset-0 -z-10"
  >
    <div
      class="-z-50 h-screen w-screen bg-orange-200"
    >
      <svg
        class="transition delay-150 duration-300 ease-in-out"
        height="100%"
        id="svg"
        preserveAspectRatio="none"
        viewBox="0 0 1000 800"
        width="100%"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          id="one"
        >
          <lineargradient
            gradientUnits="userSpaceOnUse"
            id="svg1"
            x1="-207.2916"
            x2="1991.9283"
            y1="270.9562"
            y2="270.9562"
          >
            <stop
              offset="0"
              style="stop-color: rgb(50, 222, 212);"
            />
            <stop
              offset="1"
              style="stop-color: rgb(50, 222, 212); stop-opacity: 0;"
            />
          </lineargradient>
          <path
            d="M-206.084,515.811c0,0,217.208-66.007,453.867,3c0.017,0.005,0.034,0.01,0.051,0.015 c118.716,34.611,243.627,53.152,366.838,35.887c136.417-19.115,258.875-65.218,397.701-35.334 c107.947,23.237,196.916,83.712,311.488,53.499c122.896-32.409,228.996-83.684,359.183-76.458 c111.262,6.176,199.042,51.051,308.884,10.389V-39.245h-2199.22L-206.084,515.811z"
            fill="url(#svg1)"
          />
        </g>
        <g
          id="two"
        >
          <lineargradient
            gradientUnits="userSpaceOnUse"
            id="svg2"
            x1="-306.3015"
            x2="1892.9183"
            y1="246.9538"
            y2="246.9538"
          >
            <stop
              offset="0"
              style="stop-color: rgb(50, 222, 212);"
            />
            <stop
              offset="1"
              style="stop-color: rgb(50, 222, 212); stop-opacity: 0;"
            />
          </lineargradient>
          <path
            d="M-305.094,491.808c0,0,217.208-66.007,453.867,3c0.017,0.005,0.034,0.01,0.051,0.015 c118.716,34.611,243.627,53.152,366.838,35.887c136.417-19.115,258.875-65.218,397.701-35.334 c107.946,23.237,196.916,83.712,311.488,53.499c122.896-32.409,228.996-83.684,359.183-76.458 c111.262,6.176,199.041,51.051,308.884,10.389V-63.247h-2199.22L-305.094,491.808z"
            fill="url(#svg2)"
            opacity="0.5"
          />
        </g>
        <g
          id="three"
        >
          <lineargradient
            gradientUnits="userSpaceOnUse"
            id="svg3"
            x1="-107.0737"
            x2="2090.9382"
            y1="237.9529"
            y2="237.9529"
          >
            <stop
              offset="0"
              style="stop-color: rgb(50, 222, 212);"
            />
            <stop
              offset="1"
              style="stop-color: rgb(50, 222, 212); stop-opacity: 0;"
            />
          </lineargradient>
          <path
            d="M-107.074,466.23c0,0,217.208-61.538,453.867,2.797 c0.017,0.005,0.034,0.009,0.051,0.014c118.716,32.268,243.627,49.553,366.838,33.458c136.417-17.821,258.875-60.803,397.701-32.941 c107.947,21.664,196.916,78.045,311.488,49.876c122.896-30.215,228.996-78.018,359.183-71.281 c111.262,5.758,199.042,47.594,308.884,9.686V-51.246l-2194.283,0.088L-107.074,466.23z"
            fill="url(#svg3)"
            opacity="0.5"
          />
        </g>
      </svg>
    </div>
  </div>
  <nav
    class="fixed right-0 top-0 z-20 mx-auto max-h-none w-screen"
  >
    <div
      class="absolute right-6 top-4 text-left sm:right-9 md:right-12 lg:right-48"
      data-headlessui-state=""
    >
      <div>
        <button
          aria-expanded="false"
          aria-haspopup="menu"
          class="flex items-center rounded border border-white bg-orange-200 p-2 text-black shadow-[4px_4px_0_0_rgba(60,64,43,.2)] focus:outline-none"
          data-headlessui-state=""
          id="headlessui-menu-button-:r3:"
          type="button"
        >
          <span
            class="sr-only"
          >
            Open navigation menu
          </span>
          <svg
            aria-hidden="true"
            class="h-6 w-6"
            fill="none"
            stroke="currentColor"
            stroke-width="1.5"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M3.75 5.25h16.5m-16.5 4.5h16.5m-16.5 4.5h16.5m-16.5 4.5h16.5"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </button>
      </div>
    </div>
  </nav>
  <main
    class="relative z-10 mx-auto mb-10 mt-16 h-auto max-h-none w-[90%] overflow-y-auto pb-10 pt-5 lg:w-[75%] xl:w-[60%] "
  >
    <div
      class="px-5 pb-5 pt-5 md:px-20 "
    >
      <div
        class="flex flex-col items-center justify-center py-10 text-center"
      >
        <div>
          <img
            alt="Mrugesh Mohapatra's profile picture."
            class="h-24 w-24 rounded-full border-4 border-orange-200 shadow-[5px_2px_0_0_rgba(60,64,43,.2)] backdrop-blur-lg md:h-32 md:w-32"
            id="profile-image"
            src="/images/profile.jpg"
          />
        </div>
        <div
          class="mt-6 -rotate-12 transform md:mt-20"
        >
          <h1
            class="-mt-2 rotate-12 transform text-2xl font-bold text-slate-600 md:-mt-16 md:text-4xl"
          >
            mrugesh mohapatra
          </h1>
          <div
            class="m-1 mx-auto -mt-7 w-2/5 rounded-full border-8 border-orange-200 bg-orange-200"
          />
        </div>
        <h2
          class="mt-2 max-w-md p-1 text-lg font-bold text-slate-600"
        >
          nocturnal developer ü¶â ‚Ä¢ open-source enthusiast üåè ‚Ä¢ photography noob üì∑
        </h2>
        <h3
          class="text-md mb-4 p-1 font-medium text-slate-600 "
        >
          Principal Maintainer ‚Äî Cloud Infrastructure & Open-source, 
          <a
            aria-label=""
            class="-ml-1 p-1 text-slate-600 underline decoration-orange-200 decoration-wavy decoration-2 underline-offset-2 hover:text-white hover:decoration-white"
            href="https://www.freecodecamp.org/news/team#:~:text=around%20the%20world.-,Mrugesh%20Mohapatra,-from%20Bengaluru%2C%20India"
            role="link"
            target="_blank"
            type=""
          >
            freeCodeCamp.org
          </a>
        </h3>
        <div
          class="mb-4 mt-2 flex flex-col items-center justify-center space-y-2"
        >
          <a
            aria-label=""
            class="w-80 items-center rounded-md border border-solid border-orange-50 bg-orange-400 px-1 py-1 text-sm font-medium text-black shadow-[4px_4px_0_0_rgba(60,64,43,.2)] backdrop-blur-sm hover:bg-orange-300 hover:text-slate-800 focus:outline-none focus:ring-2 focus:ring-orange-500 focus:ring-offset-2 md:px-4 md:py-2"
            href="https://topmate.io/mrugesh"
            role="link"
            target="_blank"
            type="button"
          >
            <span
              class="inline-flex items-center"
            >
              <svg
                aria-hidden="true"
                class="mr-2 h-4 w-4"
                fill="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M4.913 2.658c2.075-.27 4.19-.408 6.337-.408 2.147 0 4.262.139 6.337.408 1.922.25 3.291 1.861 3.405 3.727a4.403 4.403 0 00-1.032-.211 50.89 50.89 0 00-8.42 0c-2.358.196-4.04 2.19-4.04 4.434v4.286a4.47 4.47 0 002.433 3.984L7.28 21.53A.75.75 0 016 21v-4.03a48.527 48.527 0 01-1.087-.128C2.905 16.58 1.5 14.833 1.5 12.862V6.638c0-1.97 1.405-3.718 3.413-3.979z"
                />
                <path
                  d="M15.75 7.5c-1.376 0-2.739.057-4.086.169C10.124 7.797 9 9.103 9 10.609v4.285c0 1.507 1.128 2.814 2.67 2.94 1.243.102 2.5.157 3.768.165l2.782 2.781a.75.75 0 001.28-.53v-2.39l.33-.026c1.542-.125 2.67-1.433 2.67-2.94v-4.286c0-1.505-1.125-2.811-2.664-2.94A49.392 49.392 0 0015.75 7.5z"
                />
              </svg>
              Book a consulting session
            </span>
          </a>
          <div
            class="flex flex-row justify-between space-x-8"
          >
            <a
              aria-label=""
              class="w-36 items-center rounded-md border border-solid border-orange-50 bg-orange-200 px-2 py-1 text-sm font-medium text-slate-600 shadow-[4px_4px_0_0_rgba(60,64,43,.2)] backdrop-blur-sm hover:bg-orange-300 hover:text-slate-800 focus:outline-none focus:ring-2 focus:ring-orange-500 focus:ring-offset-2 md:px-4 md:py-2"
              href="https://hn.mrugesh.dev"
              role="link"
              target="_blank"
              type="button"
            >
              <span
                class="inline-flex items-center"
              >
                <svg
                  aria-hidden="true"
                  class="mr-2 h-3 w-3"
                  fill="currentColor"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M6.32 2.577a49.255 49.255 0 0111.36 0c1.497.174 2.57 1.46 2.57 2.93V21a.75.75 0 01-1.085.67L12 18.089l-7.165 3.583A.75.75 0 013.75 21V5.507c0-1.47 1.073-2.756 2.57-2.93z"
                    fill-rule="evenodd"
                  />
                </svg>
                Blog
              </span>
            </a>
            <a
              aria-label=""
              class="w-36 items-center rounded-md border border-solid border-orange-50 bg-orange-200 px-2 py-1 text-sm font-medium text-slate-600 shadow-[4px_4px_0_0_rgba(60,64,43,.2)] backdrop-blur-sm hover:bg-orange-300 hover:text-slate-800 focus:outline-none focus:ring-2 focus:ring-orange-500 focus:ring-offset-2 md:px-4 md:py-2"
              href="https://bio.link/mrugesh"
              role="link"
              target="_blank"
              type="button"
            >
              <span
                class="inline-flex items-center"
              >
                <svg
                  aria-hidden="true"
                  class="mr-2 h-3 w-3"
                  fill="currentColor"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M19.902 4.098a3.75 3.75 0 00-5.304 0l-4.5 4.5a3.75 3.75 0 001.035 6.037.75.75 0 01-.646 1.353 5.25 5.25 0 01-1.449-8.45l4.5-4.5a5.25 5.25 0 117.424 7.424l-1.757 1.757a.75.75 0 11-1.06-1.06l1.757-1.757a3.75 3.75 0 000-5.304zm-7.389 4.267a.75.75 0 011-.353 5.25 5.25 0 011.449 8.45l-4.5 4.5a5.25 5.25 0 11-7.424-7.424l1.757-1.757a.75.75 0 111.06 1.06l-1.757 1.757a3.75 3.75 0 105.304 5.304l4.5-4.5a3.75 3.75 0 00-1.035-6.037.75.75 0 01-.354-1z"
                    fill-rule="evenodd"
                  />
                </svg>
                Links
              </span>
            </a>
          </div>
        </div>
        <p
          class="text-slate-600-0 mx-auto p-0 text-sm font-medium text-slate-600 lg:mt-4"
        >
          Elsewhere on the internet
        </p>
        <div
          class="mx-auto mb-1 mt-2 flex flex-row items-center justify-center space-x-2 space-y-0"
        >
          <a
            aria-label=""
            class="inline-flex items-center rounded-md bg-gray-50/50 px-1 py-1 text-sm font-medium text-gray-500 shadow-[2px_2px_0_0_rgba(60,64,43,.2)] backdrop-blur-sm hover:bg-gray-500 hover:text-gray-50 focus:outline-none focus:ring-2 focus:ring-gray-500 focus:ring-offset-2 disabled:bg-gray-200 disabled:text-gray-500 md:px-4 md:py-2"
            href="https://twitter.com/raisedadead"
            role="link"
            target="_blank"
            type="button"
          >
            <span
              class="sr-only"
            >
              Twitter
            </span>
            <svg
              aria-hidden="true"
              class="h-6 w-6"
              fill="currentColor"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.29 20.251c7.547 0 11.675-6.253 11.675-11.675 0-.178 0-.355-.012-.53A8.348 8.348 0 0022 5.92a8.19 8.19 0 01-2.357.646 4.118 4.118 0 001.804-2.27 8.224 8.224 0 01-2.605.996 4.107 4.107 0 00-6.993 3.743 11.65 11.65 0 01-8.457-4.287 4.106 4.106 0 001.27 5.477A4.072 4.072 0 012.8 9.713v.052a4.105 4.105 0 003.292 4.022 4.095 4.095 0 01-1.853.07 4.108 4.108 0 003.834 2.85A8.233 8.233 0 012 18.407a11.616 11.616 0 006.29 1.84"
              />
            </svg>
          </a>
          <a
            aria-label=""
            class="inline-flex items-center rounded-md bg-gray-50/50 px-1 py-1 text-sm font-medium text-gray-500 shadow-[2px_2px_0_0_rgba(60,64,43,.2)] backdrop-blur-sm hover:bg-gray-500 hover:text-gray-50 focus:outline-none focus:ring-2 focus:ring-gray-500 focus:ring-offset-2 disabled:bg-gray-200 disabled:text-gray-500 md:px-4 md:py-2"
            href="https://github.com/raisedadead"
            role="link"
            target="_blank"
            type="button"
          >
            <span
              class="sr-only"
            >
              GitHub
            </span>
            <svg
              aria-hidden="true"
              class="h-6 w-6"
              fill="currentColor"
              viewBox="0 0 24 24"
            >
              <path
                clip-rule="evenodd"
                d="M12 2C6.477 2 2 6.484 2 12.017c0 4.425 2.865 8.18 6.839 9.504.5.092.682-.217.682-.483 0-.237-.008-.868-.013-1.703-2.782.605-3.369-1.343-3.369-1.343-.454-1.158-1.11-1.466-1.11-1.466-.908-.62.069-.608.069-.608 1.003.07 1.531 1.032 1.531 1.032.892 1.53 2.341 1.088 2.91.832.092-.647.35-1.088.636-1.338-2.22-.253-4.555-1.113-4.555-4.951 0-1.093.39-1.988 1.029-2.688-.103-.253-.446-1.272.098-2.65 0 0 .84-.27 2.75 1.026A9.564 9.564 0 0112 6.844c.85.004 1.705.115 2.504.337 1.909-1.296 2.747-1.027 2.747-1.027.546 1.379.202 2.398.1 2.651.64.7 1.028 1.595 1.028 2.688 0 3.848-2.339 4.695-4.566 4.943.359.309.678.92.678 1.855 0 1.338-.012 2.419-.012 2.747 0 .268.18.58.688.482A10.019 10.019 0 0022 12.017C22 6.484 17.522 2 12 2z"
                fill-rule="evenodd"
              />
            </svg>
          </a>
          <a
            aria-label=""
            class="inline-flex items-center rounded-md bg-gray-50/50 px-1 py-1 text-sm font-medium text-gray-500 shadow-[2px_2px_0_0_rgba(60,64,43,.2)] backdrop-blur-sm hover:bg-gray-500 hover:text-gray-50 focus:outline-none focus:ring-2 focus:ring-gray-500 focus:ring-offset-2 disabled:bg-gray-200 disabled:text-gray-500 md:px-4 md:py-2"
            href="https://instagram.com/raisedadead"
            role="link"
            target="_blank"
            type="button"
          >
            <span
              class="sr-only"
            >
              Instagram
            </span>
            <svg
              aria-hidden="true"
              class="h-6 w-6"
              fill="currentColor"
              viewBox="0 0 24 24"
            >
              <path
                clip-rule="evenodd"
                d="M12.315 2c2.43 0 2.784.013 3.808.06 1.064.049 1.791.218 2.427.465a4.902 4.902 0 011.772 1.153 4.902 4.902 0 011.153 1.772c.247.636.416 1.363.465 2.427.048 1.067.06 1.407.06 4.123v.08c0 2.643-.012 2.987-.06 4.043-.049 1.064-.218 1.791-.465 2.427a4.902 4.902 0 01-1.153 1.772 4.902 4.902 0 01-1.772 1.153c-.636.247-1.363.416-2.427.465-1.067.048-1.407.06-4.123.06h-.08c-2.643 0-2.987-.012-4.043-.06-1.064-.049-1.791-.218-2.427-.465a4.902 4.902 0 01-1.772-1.153 4.902 4.902 0 01-1.153-1.772c-.247-.636-.416-1.363-.465-2.427-.047-1.024-.06-1.379-.06-3.808v-.63c0-2.43.013-2.784.06-3.808.049-1.064.218-1.791.465-2.427a4.902 4.902 0 011.153-1.772A4.902 4.902 0 015.45 2.525c.636-.247 1.363-.416 2.427-.465C8.901 2.013 9.256 2 11.685 2h.63zm-.081 1.802h-.468c-2.456 0-2.784.011-3.807.058-.975.045-1.504.207-1.857.344-.467.182-.8.398-1.15.748-.35.35-.566.683-.748 1.15-.137.353-.3.882-.344 1.857-.047 1.023-.058 1.351-.058 3.807v.468c0 2.456.011 2.784.058 3.807.045.975.207 1.504.344 1.857.182.466.399.8.748 1.15.35.35.683.566 1.15.748.353.137.882.3 1.857.344 1.054.048 1.37.058 4.041.058h.08c2.597 0 2.917-.01 3.96-.058.976-.045 1.505-.207 1.858-.344.466-.182.8-.398 1.15-.748.35-.35.566-.683.748-1.15.137-.353.3-.882.344-1.857.048-1.055.058-1.37.058-4.041v-.08c0-2.597-.01-2.917-.058-3.96-.045-.976-.207-1.505-.344-1.858a3.097 3.097 0 00-.748-1.15 3.098 3.098 0 00-1.15-.748c-.353-.137-.882-.3-1.857-.344-1.023-.047-1.351-.058-3.807-.058zM12 6.865a5.135 5.135 0 110 10.27 5.135 5.135 0 010-10.27zm0 1.802a3.333 3.333 0 100 6.666 3.333 3.333 0 000-6.666zm5.338-3.205a1.2 1.2 0 110 2.4 1.2 1.2 0 010-2.4z"
                fill-rule="evenodd"
              />
            </svg>
          </a>
          <a
            aria-label=""
            class="inline-flex items-center rounded-md bg-gray-50/50 px-1 py-1 text-sm font-medium text-gray-500 shadow-[2px_2px_0_0_rgba(60,64,43,.2)] backdrop-blur-sm hover:bg-gray-500 hover:text-gray-50 focus:outline-none focus:ring-2 focus:ring-gray-500 focus:ring-offset-2 disabled:bg-gray-200 disabled:text-gray-500 md:px-4 md:py-2"
            href="https://linkedin.com/in/mrugeshm"
            role="link"
            target="_blank"
            type="button"
          >
            <span
              class="sr-only"
            >
              LinkedIn
            </span>
            <svg
              aria-hidden="true"
              class="h-6 w-6"
              fill="currentColor"
              viewBox="0 0 24 24"
            >
              <path
                clip-rule="evenodd"
                d="M19 0h-14c-2.761 0-5 2.239-5 5v14c0 2.761 2.239 5 5 5h14c2.762 0 5-2.239 5-5v-14c0-2.761-2.238-5-5-5zm-11 19h-3v-11h3v11zm-1.5-12.268c-.966 0-1.75-.79-1.75-1.764s.784-1.764 1.75-1.764 1.75.79 1.75 1.764-.783 1.764-1.75 1.764zm13.5 12.268h-3v-5.604c0-3.368-4-3.113-4 0v5.604h-3v-11h3v1.765c1.396-2.586 7-2.777 7 2.476v6.759z"
                fill-rule="evenodd"
              />
            </svg>
          </a>
        </div>
      </div>
    </div>
    <footer
      class="p-2"
    >
      <div
        class="font-mono text-gray-700 text-sm text-center mx-8 md:mx-auto"
      >
        <p>
          ¬© 2012-2023 Mrugesh Mohapatra Co. ‚Äî All rights reserved.
        </p>
        <p
          class="mt-2"
        >
          <a
            aria-label=""
            class="text-gray-700 hover:text-gray-50 rounded-full hover:bg-gray-500 py-1 px-2"
            href="/contact"
            role="link"
          >
            About
          </a>
          ‚Ä¢
          <a
            aria-label=""
            class="text-gray-700 hover:text-gray-50 rounded-full hover:bg-gray-500 py-1 px-2"
            href="/terms"
            role="link"
          >
            Terms
          </a>
          ‚Ä¢
          <a
            aria-label=""
            class="text-gray-700 hover:text-gray-50 rounded-full hover:bg-gray-500 py-1 px-2"
            href="/privacy"
            role="link"
          >
            Privacy
          </a>
          ‚Ä¢
          <a
            aria-label=""
            class="text-gray-700 hover:text-gray-50 rounded-full hover:bg-gray-500 py-1 px-2"
            href="/refunds"
            role="link"
          >
            Refunds
          </a>
          ‚Ä¢
          <a
            aria-label=""
            class="text-gray-700 hover:text-gray-50 rounded-full hover:bg-gray-500 py-1 px-2"
            href="/contact"
            role="link"
          >
            Contact
          </a>
        </p>
      </div>
    </footer>
  </main>
  <div
    class="fixed bottom-6 right-8 z-20 md:right-20 lg:right-48"
  />
</DocumentFragment>
`;
